jobQueue.py

```diff
@@ -134,7 +137,7 @@ def remove(self, id):
         self.log.debug("remove|Acquiring lock to job queue.")
         self.queueLock.acquire()
         self.log.debug("remove|Acquired lock to job queue.")
-        if str(id) in self.liveJobs.keys():
+        if str(id) in list(self.liveJobs.keys()):
```

Comment:
Is self.liveJobs a dictionary?

In that case it would be more suitable to change it to `if str(id) in self.liveJobs` (w/o calling keys) for python 3
instead of converting it into a list first.

https://stackoverflow.com/questions/1602934/check-if-a-given-key-already-exists-in-a-dictionary

Reply:
It's either a TangoNativeDictionary or TangoRemoteDictionary (which exports the same interface) which is not iterable so the `x in y` syntax would not work here